ledger-tool/src/bigtable.rs

```diff
@@ -648,34 +666,35 @@ impl BigTableSubCommand for App<'_, '_> {
                                 .long("force")
                                 .takes_value(false)
                                 .help(
-                                    "Force reupload of any blocks already present in BigTable instance\
-                                    Note: reupload will *not* delete any data from the tx-by-addr table;\
-                                    Use with care.",
+                                    "Force reupload of any blocks already present in BigTable \
+                                     instanceNote: reupload will *not* delete any data from the \
+                                     tx-by-addr table;Use with care.",
```

Comment:
I think those leading spaces are fine; I believe all leading whitespace the line after a `\` gets trimmed, and the extra space there is so the text is aligned, instead of slightly offset from first line that has quotation mark:
```
                                    "Force reupload of any blocks already present in BigTable \
                                     instanceNote: reupload will *not* delete any data from the \
                                     tx-by-addr table;Use with care.",
```
Note that running `solana-ledger-tool bigtable upload --help` does not show a double space:
```
$ cargo run -- bigtable upload --help
...
FLAGS:
        --force                   Force reupload of any blocks already present in BigTable instanceNote: reupload will
                                  *not* delete any data from the tx-by-addr table;Use with care.
```

Reply:
Ahh thank you, I see it now